dotcom-rendering/src/web/components/ArticleHeadline.tsx

```diff
@@ -220,36 +223,100 @@ const zIndex = css`
 	z-index: 1;
 `;
 
+const HeadlineAgeWarning = ({
+	tags,
+	webPublicationDateDeprecated,
+	isScreenReader = false,
+}: {
+	tags: TagType[];
+	webPublicationDateDeprecated: string;
+	isScreenReader?: boolean;
+}) => {
+	const ageWarningMargins = css`
+		margin-top: 12px;
+		margin-left: -10px;
+		margin-bottom: 6px;
+
+		${from.tablet} {
+			margin-left: -20px;
+		}
+
+		${from.leftCol} {
+			margin-left: -10px;
+			margin-top: 0;
+		}
+	`;
+
+	const age = getAgeWarning(tags, webPublicationDateDeprecated);
+
+	if (age && isScreenReader) {
+		return <AgeWarning age={age} isScreenReader={true} />;
+	}
+
+	if (age) {
+		return (
+			<div css={ageWarningMargins}>
+				<AgeWarning age={age} />
+			</div>
+		);
+	}
+
+	return null;
+};
+
 export const ArticleHeadline = ({
 	headlineString,
 	format,
 	tags,
 	byline,
+	webPublicationDateDeprecated,
 }: Props) => {
 	const palette = decidePalette(format);
+
 	switch (format.display) {
 		case ArticleDisplay.Immersive: {
 			switch (format.design) {
 				case ArticleDesign.PrintShop:
 					return (
 						// Immersive headlines have two versions, with main media, and (this one) without
-						<h1
-							css={[
-								jumboFont,
-								maxWidth,
-								immersiveStyles,
-								displayBlock,
-								reducedBottomPadding,
-							]}
-						>
-							{curly(headlineString)}
-						</h1>
+						<>
+							<HeadlineAgeWarning
```

Comment:
Nitpicking, worth having a wrapper for this?

Reply:
Great idea!

Done 👍